module/signature/aggregation_test.go

```diff
@@ -150,3 +150,39 @@ func BenchmarkAggregationProviderAggregation(b *testing.B) {
 		}
 	}
 }
+
+func BenchmarkVerifyMany(b *testing.B) {
+
+	// stop timer and reset to zero
+	b.StopTimer()
+	b.ResetTimer()
+
+	// create the desired number of signers
+	var signer *AggregationProvider
+	var priv crypto.PrivateKey
+	msg := createMSGB(b)
+	sigs := make([]crypto.Signature, 0, NUM_AGG_BENCH)
+	keys := make([]crypto.PublicKey, 0, NUM_AGG_BENCH)
+	for i := 0; i < NUM_AGG_BENCH; i++ {
+		signer, priv = createAggregationB(b)
+		sig, err := signer.Sign(msg)
+		if err != nil {
+			b.Fatal(err)
+		}
```

Comment:
you may replace all similar occurrences in this file with `require.NoError(b, err)`